reporter_test.go

```diff
@@ -0,0 +1,151 @@
+package check_test
+
+import (
+	"fmt"
+
+	. "gopkg.in/check.v1"
+)
+
+const testFile = "reporter_test.go"
```

Comment:
Being a bit pedantic you could get the name of the current file with something like:

```
type ReporterS struct{
  testFile string
}

func (s *ReporterS) SetUpSuite(c *C) {
  _, fileName, _, ok := runtime.Caller(0)
  c.Assert(ok, Equals, true)
  s.testFile = filepath.Base(fileName)
}
```

Moreover, this way the check_test namespace doesn't get a new identifier.